src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
+    private static final int INITIAL_QUANTITY = 0;

```

Comment:
The `INITIAL_QUANTITY` constant seems redundant as `0` is the default value returned by `getOrDefault` if the key is not present in the map. You can remove this constant and use `0` directly in `getOrDefault` calls.